### PR TITLE
Add bigquery shakespeare example

### PR DESCRIPTION
**Pull request by jonparrott**

### CODE REVIEW

**Comment by jonparrott** [comment]
@tswast thanks, currently unmergable due to https://github.com/GoogleCloudPlatform/google-cloud-python/issues/2585

**Comment by tswast** [comment]
Ugh.

File: bigquery/cloud-client/simple_app_test.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import simple_app
+
+
+def test_query_shakespeare(capsys):
+    simple_app.query_shakespeare()
+    out, _ = capsys.readouterr()
+    assert 'Hamlet' in out
```
Comment (tswast) [inline bigquery/cloud-client/simple_app_test.py]: 'hamlet' is lower-case in my query output in the Java sample I'm writing.

```
titles:
	hamlet: 5318
	kinghenryv: 5104
	cymbeline: 4875
	troilusandcressida: 4795
	kinglear: 4784
	kingrichardiii: 4713
	2kinghenryvi: 4683
	coriolanus: 4653
	2kinghenryiv: 4605
	antonyandcleopatra: 4582
total unique words: 164656
```

Comment (jonparrott) [inline bigquery/cloud-client/simple_app_test.py]: Thanks for the heads up.

**Pull request merged**